doc/user_manual_en.md

```diff
@@ -0,0 +1,111 @@
+# qTox User Manual
+## Index
+
+* [Settings](#Settings)
+* [Groupchats](#Groupchats)
+* [Multi Window Mode](#multi-window-mode)
+* [Keyboard Shortcuts](#keyboard-shortcuts)
+
+## Settings
+### General
+
+#### General Settings
+* _Language:_ Changes which language the qTox interface uses.
+* _Autostart:_ If set, qTox will start when you login on your computer.
+* _Light icon:_ If set, qTox will use a different icon, which is easier to read on black backgrounds.
+* _Show system tray icon:_ If set, qTox will show its icon in your system tray.
+    * _Start in tray:_ On start, qTox will only show its tray icon and no window.
+    * _Minimize to tray:_ The minimize button on the top right, will minimize qTox to its tray icon. There won't be a taskbar item.
+
+* _Auto away after (0 to disable):_ After the specified amount of time, qTox will set your Status to "Away". A setting of 0 will never change your status.
+* _Default directory to save files:_ You can set the directory where qTox puts files you recieved here.
+* _Autoaccept files:_ If set, qTox will automatically accept filetransfers and put them in the directory specified above.
+
+#### Chat
+* _Play sound:_ If checked, qTox will play a sound when you get a new message.
+* _Open window:_ If checked, the qTox window will be opened. <!-- TODO: Will a NEW window be opened? What happens in Multi Window mode? -->
```

Comment:
>TODO: Will a NEW window be opened?

If `Multiple windows mode` is not selected, then no. Otherwise..

>What happens in Multi Window mode?

That ~mostly depends on the `Open each chat in an individual window` option - if it's ticked on, a new window should appear.

That is, in a simple scenario where window for particular friend has not been opened before.

If it was, and it was manually grouped by user with a few other contacts ↓ , and that window wasn't closed, it should be (IIRC) brought up.
![manually grouped](https://cloud.githubusercontent.com/assets/3148759/12221499/6ed1807e-b795-11e5-80f5-137843401141.png) ← if contact is there, new window won't be opened.

If that option is ticked off, then all contacts are automatically "grouped" in that window.